File: source/code/nodejs/authorization/account_with_customdata_req.js
var accountHref = 'https://api.stormpath.com/v1/accounts/5u8BYZtu09s3yd1XtDYRSo';

client.getAccount(accountHref, {expand: 'customData'}, function (err, account) {
  if (err) {
    return console.error(err);
  }

  console.log('Custom data for account:', account.customData);
})

File: source/code/java/authorization/account_with_customdata_req.java
import com.stormpath.sdk.account;
import com.stormpath.sdk.directory;

String accountHref = "https://api.stormpath.com/v1/accounts/cJoiwcorTTmkDDBsf02AbA"
CustomData customData = client.getResource(accountHref,
                                            Account.class,
                                            Accounts.criteria().withCustomData()).getCustomData();
